shared/ssa/codeql/ssa/Ssa.qll

```diff
@@ -1369,18 +1408,6 @@ module Make<LocationSig Location, InputSig<Location> Input> {
       not uncertainWriteDefinitionInput(_, def)
     }
 
-    /** Holds if a read is not dominated by a definition. */
```

Comment:
Why was this removed?

Reply:
Ah, I was too quick there. I removed it because it referenced deprecated predicates, but I should just have updated those references. Fix incoming.